api/v1alpha1/account_types.go

```diff
@@ -176,6 +196,26 @@ func (a *Account) HasSupportCaseID() bool {
 	return a.Status.SupportCaseID != ""
 }
 
+//HasOpenQuotaIncreaseRequests returns true if an account has any open quota increase requests
+func (a *Account) HasOpenQuotaIncreaseRequests() bool {
+	for _, v := range a.Status.QuotaIncreaseRequests {
```

Comment:
Was the linter complaining about using `_, v` in general, or was it complaining about using it when the list is a list of pointers?  I don't remember, but we should double check the linter to make sure it doesn't complain about this, where it was pushing us in the past to use `for i := range myList { mylist[i].something }`